src/framework/preferences/js/ArrowScrolling.js

```diff
@@ -118,4 +134,30 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
         }
     };
 
+    fluid.prefs.arrowScrolling.getClosesPanelIndex = function (panels) {
+        var panelArray = [];
+        panels.each(function (idx, panel) {
+            panelArray.push({
+                index: idx,
+                offset: Math.abs($(panel).offset().left)
+            });
+        });
+        panelArray.sort(function (a, b) {
+            return a.offset - b.offset;
+        });
+        return panelArray[0].index;
+    };
+
+    // Based on scrollStop.js ( https://github.com/cferdinandi/scrollStop ),
+    // which is licensed under: MIT License.
+    fluid.prefs.arrowScrolling.scrollDebounce = function (elm, callback, delay) {
```

Comment:
It's unclear what this function achieves - it could do with some better commenting about its strategy and signature. Also, it seems like this would be something better done as part of the DataSource implementation for FLUID-6209. The specific piece of work that this pull request achieves doesn't seem to be written up in the JIRA, which just describes the requirement for persistence for the adjustor selection?

Reply:
Also, I think we have enough copied of this algorithm in circulation that it is probably time for it to go into the core framework - see jqUnit-node's version at https://github.com/fluid-project/node-jqunit/blob/master/lib/jqUnit-node.js#L158

Reply:
@amb26 most of the persistence came for free with work on FLUID-5534; however, that only handled the case where scrolling happened via the arrow buttons. This PR finishes the task by addressing any type of scroll action that may have happened (e.g. swiping, sliding, using the keyboard).

Reply:
I've added a [comment](https://issues.fluidproject.org/browse/FLUID-6180?focusedCommentId=37430&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-37430) to the JIRA to clarify what work this PR is covering.

Reply:
I've cherry-picked the fluid.debounce out of my fluid-6102 work and added it here. Making use of that for debouncing now.